.github/workflows/ci.yml

```
@@ -1,68 +1,85 @@
-          override: true
-      - uses: actions-rs/cargo@v1
-        with:
-          command: check
+    check:
+        name: Check
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: dtolnay/rust-toolchain@stable
+              with:
+                  profile: minimal
```

actionlint (1.7.4)

input "profile" is not defined in action "dtolnay/rust-toolchain@stable". available inputs are "components", "target", "targets", "toolchain"

(action)